packages/now-python/src/install.ts

```diff
@@ -16,10 +16,9 @@ async function isInstalled(dependency: string, cwd: string) {
 
 const makeRequirementsCheckCode = (requirementsPath: string) => `
 import distutils.text_file
-import pkg_resources
-from pkg_resources import DistributionNotFound, VersionConflict
+import importlib.util
```

Comment:
while i was testing i found that some packages take a long time to import so i found a way to check if they are installed without importing them

Reply:
Are you sure this is correct? I have a `requirements.txt` that looks like:

```
Flask==1.0.3
```

When I run `importlib.util.find_spec('Flask==1.0.3')` I get a module not found error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/importlib/util.py", line 88, in find_spec
    parent = __import__(parent_name, fromlist=['__path__'])
ModuleNotFoundError: No module named 'Flask==1'
```